cmd/engflow_auth/main.go

```diff
@@ -47,7 +47,7 @@ const (
 )
 
 type appState struct {
-	// These vars are initialized by `build()` iff they are not pre-populated;
+	// These vars are initialized by `build()` if they are not pre-populated;
```

Comment:
This wasn't a [typo](https://en.wikipedia.org/wiki/If_and_only_if), but maybe it is best to expand it fully

```suggestion
	// These vars are initialized by `build()` if and only if they are not pre-populated;
```

Reply:
huh... I've never seen that before.